### DISCUSSION TITLE
How can i run fooocus on Intel UHD Graphic 630?

### DISCUSSION THREAD

**Discussion by Jachimeq** [vladmandic/sdnext#2649] [Q&A]
I'm literal noob here, installed a fooocus, unzipped runned a bat file and after that i got an annouce of 'Found no NVIDIA driver on your system' there is a complex idea to run a program on this requirements and components?

**Comment by Jachimeq** [comment]
Also i checked a requirements of Fooocus which are 8GB RAM and 4GB VRAM and this card is good enough to run this software

**Reply by vladmandic** [reply]
You've opened an issue, I responded and then you created discussion topic?! Noob or not, this is NOT Foocus and even worse, this is really bad manners.

**Reply by Jachimeq** [reply]
I had been looking at Foocus, and on another page, I was reading about something related to your work. I don't know what happened; I ended up on the GitHub page with your software. Maybe I clicked on another referral link – I would have to check. However, after that, I will take a look at the software you made too. I hope you're not angry about that. I'm really sorry.

**Comment by brknsoul** [comment]
This git is for SD.Next. Perhaps you were looking for https://github.com/lllyasviel/Fooocus ?

Just note, Integrated GPUs (especially Intel) do not have much (if any) support in the Stable Diffusion community, simply because they aren't powerful enough to get a usable experience.
